components/filetheme.tsx

```diff
@@ -0,0 +1,98 @@
+import { useContext, useState } from "react";
+import {
+  FileThemeContext,
+  themeMap,
+  FileTheme as FileThemeType,
+} from "../system/context/FileThemeContext";
+import Image from "next/image";
+import ThemeBar from "./themebar";
+import Dialog from "./dialog";
+import MintSkin from "./mintskin";
+
+interface FileThemeProps {}
+export const FileTheme = ({}: FileThemeProps) => {
+  const { setFileTheme, fileTheme } = useContext(FileThemeContext);
+  const [testFileTheme, setTestFileTheme] = useState<FileThemeType>(fileTheme);
+
+  const [showBar, setShowBar] = useState(false);
+  const [showPayment, setShowPayment] = useState(false);
+
+  const openPayment = () => {
+    setFileTheme(testFileTheme);
+    // setShowPayment(true);
+  };
+
+  const closePayment = () => {
+    setShowPayment(false);
+  };
+
+  return (
+    <div className="px-6">
+      {showPayment && (
+        <div className="z-20">
+          <Dialog
```

Comment:
looks like `id` is an expected property on this component

Reply:
Done.